src/app/shared/navbar/navbar.scss

```diff
@@ -41,9 +41,11 @@
 }
 
 .skip-link-wrapper {
+  width: 180px;
```

Comment:
what is this `width` for?

Reply:
I think accidentally left it in there when I was trying something